scan/libmailgoose/translate.py

```diff
@@ -502,6 +502,12 @@
             "Zgodnie ze specyfikacją SPF, nie należy używać mechanizmu 'ptr'. Pod adresem "
             "https://tools.ietf.org/html/rfc7208#section-5.5 można znaleźć uzasadnienie tej rekomendacji.",
         ),
+        (
+            f"The IP address {PLACEHOLDER} from which the message was received "
+            f"is not authorized by the domain's policy to send messages.",
+            f"Adres IP {PLACEHOLDER}, z którego otrzymano wiadomość, nie jest "
+            f"uprawniony przez politykę domeny do wysyłania wiadomości.",
```

Comment:
przez politykę SPF warto dodac ze

Reply:
This error message is displayed in the SPF section, so I believe that adding SPF to the error message is redundant.